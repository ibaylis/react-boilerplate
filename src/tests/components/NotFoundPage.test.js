import React from 'react';
import { shallow } from 'enzyme';
import expenses from '../fixtures/expenses';
import NotFoundPage from '../../components/NotFoundPage';


test('should render Expense Dashboard Page correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot()
})