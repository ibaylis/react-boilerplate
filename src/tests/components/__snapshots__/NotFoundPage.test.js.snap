// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense Dashboard Page correctly 1`] = `
<div>
  404! Not Found 
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
